.circleci/config.yml

```diff
@@ -1277,3 +1307,9 @@ workflows:
               only: /^v20.*/
             branches:
               ignore: /.*/
+      - publish_docs:
+          filters:
+            tags:
+              only: /^v20.*/
+            branches:
+              ignore: /.*/
```

Comment:
Lot of copy & paste of these filters… might make sense to share them and `<<` include them.